internal/tools/tools.go

```diff
@@ -15,10 +15,12 @@ import (
 	_ "github.com/frapposelli/wwhrd"
 	_ "github.com/fzipp/gocyclo"
 	_ "github.com/go-enry/go-license-detector/v4/cmd/license-detector"
+	_ "github.com/golang/mock/mockgen"
 	_ "github.com/golangci/golangci-lint/cmd/golangci-lint"
 	_ "github.com/gordonklaus/ineffassign"
 	_ "github.com/goware/modvendor"
 	_ "github.com/mgechev/revive"
+	_ "github.com/stormcat24/protodep"
```

Comment:
I am fine leaving this here but I am curious if there is a reason for it to be here instead of on the `proto` module

Reply:
So we get the latest `protodep`, otherwise the pins in `internal/tools/proto` create issues.